test/compile_test/instantiate.hpp

```diff
@@ -538,7 +618,18 @@ void instantiate(RealType)
    boost::math::sinc_pi(v1 * 1);
    boost::math::sinhc_pi(v1 * 1);
    boost::math::asinh(v1 * 1);
-   boost::math::acosh(v1 * 1);
+   {
+      using std::fabs;
+
+      auto v_special { v1 };
```

Comment:
As above, specific value.

Reply:
There are a bunch of those. I'm more concerned with how to get the exceptions and the instantiations running and compiling.